package.json

```diff
@@ -13,10 +13,12 @@
     "@fortawesome/fontawesome-free": "^6.5.1",
     "axios": "^1.6.7",
     "bootstrap": "^5.3.3",
+    "chart.js": "^4.4.2",
```

Comment:
I noticed that we installed two packages, do we need both? if we need only one, consider removing the other.